rules-tests/CodingStyle/Rector/Encapsed/EncapsedStringsToSprintfRector/Fixture/encapsed_strings_to_sprintf_should_escape_percent.php.inc

```diff
@@ -19,7 +19,7 @@ final class EncapsedStringsToSprintfShouldEscapePercent
 {
     public function run(string $value)
     {
-        return sprintf('%s%', $value);
+        return sprintf('%s%%', $value);
```

Comment:
This now removes former test fixture. Could you add a new file so we have them both?

Reply:
But former test fixture was originally like this, Ref: rectorphp/rector#5176.

Pattern `%s%` is not valid.

Reply:
A new file added.

Reply:
So the old fixture should not work?
If so, then it's ok to have just one. I thought the old fixture is correct

Reply:
I just removed the new file and updated the old fixture.